### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,19 @@ The directory is used by [Remix IDE](https://remix.ethereum.org) for loading the
...
+
+ - The plugin should work on both light and dark theme types. The best would be to check them before approving any change or any new plugin.
+ - No `svg` file is acceptable as a plugin icon for security reasons. We recommend using `webp` or png formats.
+ - Profile `json` should contain as much info as possible. Having a short, reasonable description as well as the link to docs and the repo will inprove the onboarding of users
+ - maintained by Remix should be added to plugins 100% controlled by the Remix team only.
+   
  # Community
  
- Feel free to join our gitter chat if you have any question
+ Feel free to join our gitter chat if you have any questions
```
Comment (Aniket-Engg) [inline README.md:20]: We seems no more active on gitter. Can you put discord link here?

File: README.md [hunk 2]
```diff
@@ -5,12 +5,19 @@ The directory is used by [Remix IDE](https://remix.ethereum.org) for loading the
...
-Read all about getting started with plugin development and guidelines how to make it work for you 
+Read all about getting started with plugin development and guidelines on how to make it work for you 
 <a href='https://remix-plugins-directory.readthedocs.io/en/latest/' target=_blank>https://remix-plugins-directory.readthedocs.io/en/latest/</a>
- 
+
+ # Conditions for adding a new plugin
+
+ - The plugin should work on both light and dark theme types. The best would be to check them before approving any change or any new plugin.
+ - No `svg` file is acceptable as a plugin icon for security reasons. We recommend using `webp` or png formats.
+ - Profile `json` should contain as much info as possible. Having a short, reasonable description as well as the link to docs and the repo will inprove the onboarding of users
```
Comment (Aniket-Engg) [inline README.md:15]: typo in `inprove`

File: README.md [hunk 3]
```diff
@@ -5,12 +5,19 @@ The directory is used by [Remix IDE](https://remix.ethereum.org) for loading the
...
+ - maintained by Remix should be added to plugins 100% controlled by the Remix team only.
```
Comment (Aniket-Engg) [inline README.md:16]: `m` should be captialized in `maintained`

**Review by Aniket-Engg — APPROVED** [review]

**Comment by BENBELLA23** [comment]
https://codespaces.new/ethereum/remix-plugins-directory/pull/357?quickstart=1

**Pull request merged**